src/button-dropdown/analytics-metadata/interfaces.ts

```diff
@@ -1,6 +1,8 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 
+import { LabelIdentifier } from '@cloudscape-design/component-toolkit/internal/analytics-metadata/interfaces';
```

Comment:
This one is not compatible with `exports` declaration in the toolkit package

Reply:
I have already introduced it in other places, will fix the export, but won't manage to propagate it in time

Reply:
I would rather merge this now, and only update the import later

Reply:
PR ready in component-toolkit: https://github.com/cloudscape-design/component-toolkit/pull/85

Reply:
Exposing `internal/analytics-metadata/interfaces` as one more export looks like mess. I think we need to re-export all features from `internal/analytics-metadata` instead

For the timeline, if you say there are already instances of this reached live, we can merge this too.

But if not, we will have to put this on hold until the value is exported properly

Reply:
https://github.com/search?q=repo%3Acloudscape-design%2Fcomponents%20%40cloudscape-design%2Fcomponent-toolkit%2Finternal%2Fanalytics-metadata%2Finterfaces&type=code

But none reached live I believe

Reply:
Discussed in DM, this can be merged after the related changes in component-toolkit are released